contrib/kube-nodered/README.md

```diff
@@ -0,0 +1,324 @@
+# Kube-node-red(en)
+[![hackmd-github-sync-badge](https://hackmd.io/cocSOGQMR-qzo7DHdwgRsQ/badge)](https://hackmd.io/cocSOGQMR-qzo7DHdwgRsQ)
+
+Kube-node-red is aiming to integrate Kubeflow/Kubebeters with node-red, leveraging node-red's low-code modules, and using Kubeflow resources (e.g. Kubeflow pipeline, Kserve) to enhance its AI/ML ability.
+## Architecture
+![5A0ECFB3-D5AC-4A89-8AD5-14696A9E0449](https://github.com/NightLightTw/kubeflow-Node-RED/assets/78789817/7cce84cf-a4df-47a6-9992-9412bc70819b)
+## Demo
+[![demo](https://i.ytimg.com/vi/72tXYl6FcvU/hqdefault.jpg)](https://youtu.be/72tXYl6FcvU)
+
+## Reference
+https://github.com/NightLightTw/kubeflow-Node-RED/tree/main
+
+https://github.com/kubeflow/pipelines/tree/1.8.21/backend/api/python_http_client
+
+[Kubeflow-Node-red implementation Random Forest training flo](https://hackmd.io/@Nhi7So-lTz2m5R6pHyCLcA/Sk1eZFTbh)
+
+
+# Usage instruction
+## Prerequsite
+
+As this project focused on the node-red integration with Kubeflow, one running Kubeflow instance should be ready on a publicly available network.
+(If you need to provision your own Kubeflow instance, you could refer to our [mulitkf](https://github.com/footprintai/multikf) project to allocate one instance for developing.)
+
+## Build Container Image
+
+You can proceed to [examples](./examples/README.md) folder and run the `./build.sh` to build the image locally or just run the `./run.sh` which would download the base image from our public repository.
```

Comment:
Please use a grammar checker. For example it should be "You can proceed to THE [examples](./examples/README.md) folder"